### PR TITLE
3.4 istio doc update for openshift mesh certification

### PR DESCRIPTION
**Pull request by jshum2479**

### CODE REVIEW

File: documentation/3.4/content/userguide/istio/istio.md [hunk 1]
```diff
@@ -34,6 +34,10 @@ description: "Lets you run the operator, and WebLogic domains managed by the ope
 These instructions assume that you are using a Kubernetes cluster with
 [Istio](https://istio.io/latest/docs/setup/install/) installed and configured already.  The operator will not install
 Istio for you.
+
+For Red Hat OpenShift Service Mesh which has its own implementation of Istio, refer to the correct version of the documentation for installation.
```
Comment (rosemarymarano) [inline documentation/3.4/content/userguide/istio/istio.md:38]: Mesh which -> Mesh, (comma) which

File: documentation/3.4/content/userguide/istio/istio.md [hunk 2]
```diff
@@ -43,9 +47,18 @@ If your applications have suitable tracing code in them, then you will also be a
 use distributed tracing, such as Jaeger, to trace requests across domains and to
 other components and services that have tracing enabled.
 
+WebLogic Kubernetes Operator assume you are familiar with Istio or Red Hat OpenShift Service Mesh. If you are new to
```
Comment (rosemarymarano) [inline documentation/3.4/content/userguide/istio/istio.md:50]: assume you -> assumes that you

File: documentation/3.4/content/userguide/istio/istio.md [hunk 3]
```diff
@@ -43,9 +47,18 @@ If your applications have suitable tracing code in them, then you will also be a
...
+them, we strongly recommend reading the respective documentations and work through the `Bookinfo`
```
Comment (rosemarymarano) [inline documentation/3.4/content/userguide/istio/istio.md:51]: documentations and work -> documentation and working

File: documentation/3.4/content/userguide/istio/istio.md [hunk 4]
```diff
@@ -43,9 +47,18 @@ If your applications have suitable tracing code in them, then you will also be a
...
+sample application to familiar with the mesh and verify it is working properly in your environment before proceeding working with the Operator.
```
Comment (rosemarymarano) [inline documentation/3.4/content/userguide/istio/istio.md:52]: to familiar -> to familiarize yourself
verify it -> verify that it
environment  -> environment, (comma)
proceeding working with the Operator -> proceeding to work with the operator

File: documentation/3.4/content/userguide/istio/istio.md [hunk 5]
```diff
@@ -43,9 +47,18 @@ If your applications have suitable tracing code in them, then you will also be a
...
+
+
 To learn more about Istio,
 see [What is Istio](https://istio.io/latest/docs/concepts/what-is-istio/).  
 
+For Redhat OpenShift Service Mesh, see
+
+[OpenShift Service Mesh](https://docs.openshift.com/container-platform/4.10/service_mesh/v2x/ossm-about.html)
```
Comment (rosemarymarano) [inline documentation/3.4/content/userguide/istio/istio.md:60]: Period at the end of the sentence.

File: documentation/3.4/content/userguide/istio/istio.md [hunk 6]
```diff
@@ -54,6 +67,8 @@ The current support for Istio has these limitations:
   and has been tested with single and multicluster
   Istio installations from 1.7.3 up to 1.11.x.
 
+* For Redhat OpenShift Service Mesh,  we have tested with Red Hat OpenShift Container Platform version 4.10.20, Service Mesh 2.2.1
```
Comment (rosemarymarano) [inline documentation/3.4/content/userguide/istio/istio.md:70]: Period at the end of the sentence.

**Review by rosemarymarano — COMMENTED** [review]
Few edits

**Comment by anpanigr** [comment]
Should we mention about the istio configuration setting related to  localhostBindingsEnabled ?
In the above certified configuration the localhostBindingsEnabled MUST be set to true

**Comment by jshum2479** [comment]
@anpanigr   It is there line 206.   

  **For Redhat OpenShift Service Mesh**,  you do not need to set `istioLocalhostBindingsEnabled` or if set, it must be set to `true`.

**Review by rosemarymarano — APPROVED** [review]
Just missed one edit on line 51, documentations and work -> documentation and working
otherwise ok.

**Review by rjeberhard — APPROVED** [review]

**Pull request merged**